AaSystem/RequestManagment/CommandFileReferences.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+import Agent.ProcessControl.ProcessCommands
+import Agent.SystemCommands.AaSystemCommands
+import Agent.BatchCommands.BatchCommandHandler
+import AaSystem.AaSystemControl.AaUpgrade
+import Agent.BashCommands.Bash_AdministrativeCommands
+import Agent.BashCommands.Bash_ChangeMachineFilesAndSettings
+import Agent.BashCommands.Bash_CustomCommand
+import Agent.BashCommands.Bash_FileRetrieval
+import Agent.BashCommands.Bash_InformationRetreivingCommands
+import Agent.BashCommands.Bash_ProcessAndPortCommands
+
+
+CommandTypes = {
```

Comment:
what about Factory design pattern?

Reply:
What about it? This is dynamic discovery of public api functions.
There is not instantiation since their all static atm